### PR TITLE
docs: Corrections and formatting

### PR DESCRIPTION
**Pull request by elliotxjones**
- Formats commands and file paths as code.
- Re-words grammar for improved readability.
- Fixes QPL config steps in sect 3.3 to conform to actual config syntax.
- Adds section header links where referenced.
- Formats link test to active links.
- Updates nodejs install steps in sect 4.2 to use v18 advised by deprecation warning.
- Adds notice for missing pm2 command.

### CODE REVIEW

**Comment by elliotxjones** [comment]
Updated PR with corrected sign-off.

**Review by mingweishih — APPROVED** [review]

**Comment by elliotxjones** [comment]
> Hello @elliotxjones, thank you for your contribution of updating and improving our documentation. These are great changes and I only have a few minor changes to request.
> 
> Also, I'm interested in whether you were able to set up nodejs following the deprecation of the nodesource setup scripts?

Yes, I was able to set up nodejs just fine using v18. I was unable to continue the setup using v13 due to the deprecation warning.

File: docs/GettingStartedDocs/Contributors/NonAccMachineSGXLinuxGettingStarted.md [hunk 1]
```diff
@@ -286,18 +299,18 @@ Total Test time (real) =  83.61 sec
 
 A clean pass of the above unit tests is an indication that your Open Enclave setup was successful.
 
-Above remote attestation tests/samples will only succeed if using a production CPU. 
-If you want remote attestation tests/samples to succeed on the pre-production CPU, you should follow below steps to switch backend provisioning server to pre-production.
+The remote attestation tests/samples aboce will only succeed when using a production CPU.
```
[the remark this answers is not in the document]
Comment (elliotxjones) [reply docs/GettingStartedDocs/Contributors/NonAccMachineSGXLinuxGettingStarted.md:302]: Oops! Glad you caught that. I've amended the commit with the change.

**Review by CyanDevs — APPROVED** [review]
Thanks for making the changes!

**Pull request merged**